### DISCUSSION TITLE
API Endpoint not active when running as Windows Service

### DISCUSSION THREAD

**Discussion by xgcssch** [vectordotdev/vector#6068] [Q&A] [answered]
vector 0.11.1 (v0.11.1 x86_64-pc-windows-msvc 2020-12-17)

Hello!

I have specified

```
[api]
  enabled = true
```
in the configuration.
When i start vector from a Command Prompt, the Endpoint is accessible.
When started as a Windows Service, the Endpoint is not available.

Is this a problem with the configuration, or a problem?

Cheers 
Sönke

**Comment by jszwedko** [comment, marked as answer]
Thanks for raising this up @xgcssch . This seems like a bug to me. I opened an issue over here to track: https://github.com/timberio/vector/issues/6187
